### ISSUE TITLE
add openssl-c_rehash to the base image

### ISSUE THREAD

**Issue by fabianbrash** [vmware/photon#896]
I  would like to see openssl-c_rehash added as a package in the base image of photon OS, as most organizations are doing TLS decryption this package is extremely important so we don't have the chicken and egg scenario, it would be nice if this was included so I don't have to do weird hacks to install it, then I have to revert the weird hacks because now I've added the new root CA(s)

**Comment by suezzelur** [comment]
@fabianbrash are you not able to do a tdnf install from within your org?

**Comment by fabianbrash** [comment]
yes but I have to change /etc/yum.repos and remove https to http

**Comment by MnrGreg** [comment]
`rehash_ca_certificates.sh` is also not widely known. Could we consider symlink'ing it to `update-ca-trust`. At a high level, it performs the same function.

A number of other boot strap tools such as `cloud-init` would then work as expected.

https://www.linux.org/docs/man8/update-ca-trust.html

**Comment by spkane** [comment]
@suezzelur Would it be possible to get some additional thoughts from a maintainer or contributor, such as yourself?
